httpie/core.py

```diff
@@ -131,42 +142,68 @@ def program(args, env, log_error):
                 'outfile': env.stdout,
                 'flush': env.stdout_isatty or args.stream
             }
+            if response is final_response:
+                has_branched('program', 6)
+            else:
+                has_branched('program', 7)
             try:
-                if env.is_windows and is_py3 and 'colors' in args.prettify:
-                    write_stream_with_colors_win_py3(**write_stream_kwargs)
-                else:
+                if env.is_windows:
+                    has_branched('program', 8)
+                    if is_py3:
+                        has_branched('program', 9)
+                        if 'colors' in args.prettify:
+                            has_branched('program', 10)
+                            write_stream_with_colors_win_py3(**write_stream_kwargs)
+                elif not env.is_windows or not is_py3 or not 'colors' in args.prettify:
+                    has_branched('program', 11)
                     write_stream(**write_stream_kwargs)
             except IOError as e:
-                if not show_traceback and e.errno == errno.EPIPE:
-                    # Ignore broken pipes unless --traceback.
-                    env.stderr.write('\n')
-                else:
+                has_branched('program', 12)
+                if not show_traceback:
+                    has_branched('program', 13)
+                    if e.errno == errno.EPIPE:
+                        has_branched('program', 14)
+                        # Ignore broken pipes unless --traceback.
+                        env.stderr.write('\n')
+                elif show_traceback or not e.errno == errno.EPIPE:
```

Comment:
This didn't fully fix the issue, since we won't get here if the first clause is true but the second one is not. 
That said, I'm not sure we can put more time on it.